src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetInstaller.cs

```diff
@@ -82,33 +105,35 @@ public async Task<InstallResult> InstallAsync(InstallRequest installRequest)
             }
         }
 
-        public Task<UninstallResult> UninstallAsync(IManagedTemplatesSource managedSource)
+        public (string mountPointUri, IReadOnlyDictionary<string, string> details) Serialize(IManagedTemplatesSource managedSource)
         {
-            //Remove managedSource.MountPointUri
-            throw new NotImplementedException();
+            return (managedSource.MountPointUri, managedSource.Details);
         }
 
-        public Task<IReadOnlyList<ManagedTemplatesSourceUpdate>> GetLatestVersionAsync(IEnumerable<IManagedTemplatesSource> sources)
+        public Task<UninstallResult> UninstallAsync(IManagedTemplatesSource managedSource)
```

Comment:
This and similar methods in the same interface could use a cancellation token, unless we are worried about changing the interface.